clients/admin-ui/src/features/data-discovery-and-detection/discovery-detection.slice.ts

```diff
@@ -69,7 +73,10 @@ const discoveryDetectionApi = baseApi.injectEndpoints({
       query: (body) => ({
         method: "PUT",
         url: `/plus/discovery-monitor`,
-        body,
+        body: {
+          ...body,
+          last_monitored: undefined,
```

Comment:
`last_monitored` is read-only and shouldn't be sent in a request, see [https://ethyca.atlassian.net/browse/PROD-2145?focusedCommentId=41803](here) for context.

Reply:
could we actually make this a code comment?